### ISSUE TITLE
NTR: Flow cytometry: FOXP3-expressing T-cells decreased (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8721]
New term request Flow cytometry: FOXP3-expressing T-cells decreased (elthat_100502213109):

**Comment by pnrobinson** [comment]
@MickeySegal
this seems to be a research-only finding for https://omim.org/entry/304790, in which case it would be out of scope. If this is used for current clinical investigations let's include it. Thoughts?

**Comment by MickeySegal** [comment]
Listed in https://www.ncbi.nlm.nih.gov/books/NBK1118/ "Diagnosis/testing.  The diagnosis is established in a male proband with typical clinical findings, absent regulatory T cells (Treg) in blood or tissues, **decreased numbers of FOXP3-expressing T cells in peripheral blood determined by flow cytometry**"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000473
- label: Decreased FOXP3-expressing T cell count
- parents: [Abnormal T cell morphology (HP:0002843)](https://hpo.jax.org/app/browse/term/HP:0002843)
- pmid: PMID:20301297
- def: The number of T cells in peripheral blood that express FOXP3-expressing is below the lower limit of normal. This finding is determined by flow cytometry.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8721
- synonyms:
